packages/go_router_builder/tool/run_tests.dart

```diff
@@ -56,6 +56,6 @@ Future<void> main() async {
           .trim()
           .replaceAll('\r\n', '\n');
       expect(generated, equals(expectResult.replaceAll('\r\n', '\n')));
-    });
+    }, timeout: const Timeout(Duration(seconds: 100)));
```

Comment:
I think the default is 30 seconds, is it running longer than that?

Reply:
yes,I added this to avoid timing out